### ISSUE TITLE
Problem mit eigenen CSS-Formatierungen

### ISSUE THREAD

**Issue by Stefko** [cliffparnitzky/TinyMceFontAwesome#8]
In meiner be_tinyMCE.html5 lade ich einige eigene CSS-Formatierungen per
,content_css: ['/files/Kickstart/tinymce.css']
Klappt soweit.
Wenn ich nun tiny-mce-font-awesome installiere erhalte ich im Editor die Fehlermeldung
Failed to load content css: http://meineDomain.de/'/files/Kickstart/tinymce.css
Man beachte das /'/ in dem Pfad.
Nach der Deinstallation der Erweiterung klappt wieder alles.
Any Idea?

**Comment by cliffparnitzky** [comment]
Ne Ahnung direkt hab ich nicht. Mal so auf die Schnelle:

Versuche mal so `content_css: "/files/Kickstart/tinymce.css"`.

Also ohne eckige Klammern und mit doppelten Anführungszeichen.

**Comment by Stefko** [comment]
Bringt leider auch nichts, außer dass das /'/ verschwindet. Es heißt weiterhin:
Failed to load content css: http://meineDomain.de/files/Kickstart/tinymce.css
Die Datei ist unter dieser Adresse aber auch im Browser aufrufbar.

**Comment by cliffparnitzky** [comment]
Hmm, dann weiß ich es so erstmal nicht. Müsste man nachbauen und schauen, was final im Quellcode in die Tiny Config geschrieben wird.

**Comment by fritzmg** [comment]
The extension seems to produce invalid output, e.g.:

```
content_css: "system/themes/flexible/tinymce.min.css',
,//netdna.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
```

**Comment by tpk109** [comment]
After an update from Contao 4.9 to 4.13 I also hat the problem with the font-awesome link. It produced an error and the editor did not work anymore. Had to uninstall the extension.

**Comment by cliffparnitzky** [comment]
Ja, mit Contao 4.13 geht sowie kaum ein Plugin: https://github.com/cliffparnitzky/TinyMcePluginLoader#plugins
